tests/drivers/adc/adc_api/boards/rpi_pico_w.overlay

```diff
@@ -0,0 +1 @@
+rpi_pico.overlay
```

Comment:
We're not usually create link in repository.
Please create copy of rpi_pico.overlay.
(Also, Don't forget rewrite copyright by your name.)

Reply:
Done.